caffeine/src/main/java/com/percolate/caffeine/DialogUtils.java

```diff
@@ -28,15 +31,16 @@
      * @return AlertDialog that is being displayed.
      */
     public static AlertDialog quickDialog(final Activity context, final String message) {
+        if(dialog != null && dialog.isShowing()) return null;
```

Comment:
@brentwatson I felt uneasy about it Lol. Thats why I was trying to explain in the morning but it was not coming out right 😓. But actually your second idea is what I think is best, if anything it will be the dialog they want. If we return the original dialog it may be confusing bug to find if this causes any breaks on anyone's projects.